# 2022-01-20 07:04:15.439185: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 AVX512F FMATo enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.2022-01-20 07:04:15.441501: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.Traceback (most recent call last):  File "/home/neo/mlfuzz/tensorflow/synthesized/all-crashes/all/GatherNd_a2c326d5eb683f27426b1e6df71e3beb.py", line 5, in <module>    tf.raw_ops.GatherNd(params=arg_0, indices=arg_1)  File "/home/neo/anaconda3/envs/tf/lib/python3.9/site-packages/tensorflow/python/util/tf_export.py", line 404, in wrapper    return f(**kwargs)  File "/home/neo/anaconda3/envs/tf/lib/python3.9/site-packages/tensorflow/python/ops/gen_array_ops.py", line 3715, in gather_nd    _ops.raise_from_not_ok_status(e, name)  File "/home/neo/anaconda3/envs/tf/lib/python3.9/site-packages/tensorflow/python/framework/ops.py", line 6941, in raise_from_not_ok_status    six.raise_from(core._status_to_exception(e.code, message), None)  File "<string>", line 3, in raise_fromtensorflow.python.framework.errors_impl.InvalidArgumentError: Value for attr 'Tindices' of double is not in the list of allowed values: int32, int64	; NodeDef: {{node GatherNd}}; Op<name=GatherNd; signature=params:Tparams, indices:Tindices -> output:Tparams; attr=Tparams:type; attr=Tindices:type,allowed=[DT_INT32, DT_INT64]> [Op:GatherNd]

import tensorflow as tf

arg_0 = tf.constant(0, shape=[5], dtype=tf.float32)
arg_1 = tf.constant([], shape=[0], dtype=tf.float64)
tf.raw_ops.GatherNd(params=arg_0, indices=arg_1)